### PR TITLE
front: fix spinner

### LINKED ISSUE #3095: front: fix spinner
- fix spinner

### PR DESCRIPTION
**Pull request by clarani**
Closes #3095

### CODE REVIEW

File: front/src/common/Pathfinding/Pathfinding.tsx [hunk 1]
```diff
@@ -339,8 +340,8 @@ function Pathfinding({ zoomToFeature }: PathfindingProps) {
     <div className="pathfinding-in-progress">
       <div className="pathfinding-in-progress-card">
         <div className="pathfinding-in-progress-title-container">
-          <div className="spinner-border" role="status">
-            <span className="sr-only">Loading...</span>
+          <div>
+            <Spinner />
           </div>
```
Comment (anisometropie) [inline front/src/common/Pathfinding/Pathfinding.tsx]:
```suggestion
          <Spinner />
```

File: front/src/applications/editor/components/EntitySumUp.tsx [hunk 2]
```diff
@@ -217,12 +218,7 @@ const EntitySumUp: FC<
     }
   }, [entity, id, objType, osrdConf.infraID, state.type]);
 
-  if (state.type === 'loading' || state.type === 'idle')
-    return (
-      <div className="spinner-border" role="status">
-        <span className="sr-only">Loading...</span>
-      </div>
-    );
+  if (state.type === 'loading' || state.type === 'idle') return <Spinner />;
```
Comment (anisometropie) [inline front/src/applications/editor/components/EntitySumUp.tsx:221]:
```suggestion
  if (['loading', 'idle'].includes(state.type)) return <Spinner />;
```

**Review by anisometropie — CHANGES_REQUESTED** [review]

File: front/src/applications/editor/components/EntitySumUp.tsx (hunk 2 above)
Comment (clarani) [reply front/src/applications/editor/components/EntitySumUp.tsx:221]: I tried to change this but it's better if we keep this line like this for typescript. You can call me if you want to discuss more

File: front/src/common/Pathfinding/Pathfinding.tsx (hunk 1 above)
Comment (clarani) [reply front/src/common/Pathfinding/Pathfinding.tsx]: This div is needed, otherwise the spinner is not displayed correctly

Comment (clarani) [reply front/src/common/Pathfinding/Pathfinding.tsx]: Fixed as you asked me ! :)

**Review by Math-R — DISMISSED** [review]

**Review by Math-R — APPROVED** [review]

**Review by anisometropie — APPROVED** [review]

**Pull request merged**